pkg/sql/schemachanger/scpb/scpb.proto

```diff
@@ -182,10 +183,32 @@ message Type {
 
 message Schema {
   uint32 schema_id = 1 [(gogoproto.customname) = "SchemaID", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
-  repeated uint32 dependentObjects = 2  [(gogoproto.customname) = "DependentObjects", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+  repeated uint32 dependentObjects = 3  [(gogoproto.customname) = "DependentObjects", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
 }
 
 message Database {
   uint32 database_id = 1 [(gogoproto.customname) = "DatabaseID", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
-  repeated uint32 dependentObjects = 2  [(gogoproto.customname) = "DependentObjects", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+  repeated uint32 dependentObjects = 3  [(gogoproto.customname) = "DependentObjects", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+}
+
+message Authorization {
+  string username  = 1;
+  string appName = 2 [(gogoproto.customname) = "AppName"];
+}
+
+message Statement {
+  string statement = 1;
+}
+
+message TargetMetaData {
+  uint32 sub_work_id =  1[(gogoproto.customname) = "SubWorkID"];
+  uint32 source_element_id = 2 [(gogoproto.customname) = "SourceElementID", (gogoproto.casttype) = "SourceElementID"];
+  uint32 statement_id = 3 [(gogoproto.customname) = "StatementID"];
+}
+
+message ElementMetaData {
+  TargetMetaData common = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
+  string username  = 2;
+  string appName = 3 [(gogoproto.customname) = "AppName"];
+  string statement = 4;
```

Comment:
Given the `TargetMetadata` has the `StatementID`, it seems odd to repeat the statement here.